### PR TITLE
removed superfluous response type definition in swagger docs

### PR DESCRIPTION
**Pull request by mzedel**
Changelog: None
Signed-off-by: Manuel Zedel <manuel.zedel@northern.tech>

### CODE REVIEW

**Comment by mchalski** [comment]
@mzedel we tried that before and it failed spectacularly for the rendered docs (or something - something along the pipeline failed competely). did anything change along the way?

File: docs/management_api.yml [hunk 1]
```diff
@@ -472,10 +472,9 @@ definitions:
     example:
       application/json:
```
Comment (mzedel) [inline docs/management_api.yml]: @mchalski that seems to be required

**Comment by mzedel** [comment]
removing the outer `application/json` seems to be problematic, this one is basically an outlier in the responses

File: docs/management_api.yml (hunk 1 above)
Comment (mchalski) [reply docs/management_api.yml]: oh, oh... I can see it now. approved - if you're sure it renders ok .

**Pull request merged**